code/modules/power/engines/supermatter/supermatter.dm

```diff
@@ -573,6 +573,9 @@
 
 		if(produces_gas)
 			env.merge(removed)
+			if(istype(T, /turf/simulated))
```

Comment:
```suggestion
			if(issimulatedturf(T))
```
We have a selection is istype macros in the ``is_helpers`` file so we don't need to manually istype this stuff